### PR TITLE
[hi] Add content/hi/case-studies/_index.md

### PR DESCRIPTION
**Pull request by harshmathurx**
Hello!

 As discussed in #30731 
I have translated the `content/en/case-studies/_index.md` and added the translation to `content/hi/case-studies/_index.md`

### CODE REVIEW

**Review by anubha-v-ardhan — COMMENTED** [review]
Extremely sorry for wrong info on issue but the file here should be `.html` and not `.md`. I have updated the issue to say so too.

Can you update it to html

**Comment by harshmathurx** [comment]
> Extremely sorry for wrong info on issue but the file here should be `.html` and not `.md`. I have updated the issue to say so too.
> 
> Can you update it to html

It's alright :)
I made the changes

File: content/hi/case-studies/_index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+title: वैयक्तिक अध्ययन
+linkTitle: वैयक्तिक अध्ययन
+bigheader: कुबेरनेट्स उपयोगकर्ता वैयक्तिक अध्ययन
+abstract: उत्पादन में कुबेरनेट्स चलाने वाले उपयोगकर्ताओं का एक संग्रह.
```
Comment (anubha-v-ardhan) [inline content/hi/case-studies/_index.html:5]:
```suggestion
abstract: उत्पादन में कुबेरनेट्स उपयोग करने वाले उपयोगकर्ताओं का एक संग्रह।
```

File: content/hi/case-studies/_index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+layout: basic
+class: gridPage
+cid: caseStudies
+---
+
```
Comment (anubha-v-ardhan) [inline content/hi/case-studies/_index.html:10]:
```suggestion
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Review by anubha-v-ardhan — COMMENTED** [review]
/lgtm
/approve

Thank you @harshmathurx

**Pull request merged**